### PR TITLE
jws onboarding on ibmcloud

### PR DESCRIPTION
**Pull request by smatula**

### CODE REVIEW

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-jws-qe-interop-ocp-ci-main-ocp-4.14-lp-interop-jws-interop-ibmcloud periodic-ci-jws-qe-interop-ocp-ci-main-ocp-4.15-lp-interop-jws-interop-ibmcloud

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-jws-qe-interop-ocp-ci-main-ocp-4.14-lp-interop-jws-interop-ibmcloud periodic-ci-jws-qe-interop-ocp-ci-main-ocp-4.15-lp-interop-jws-interop-ibmcloud

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-jws-qe-interop-ocp-ci-main-ocp-4.14-lp-interop-jws-interop-ibmcloud

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-jws-qe-interop-ocp-ci-main-ocp-4.15-lp-interop-jws-interop-ibmcloud

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-jws-qe-interop-ocp-ci-main-ocp-4.14-lp-interop-jws-interop-ibmcloud

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-jws-qe-interop-ocp-ci-main-ocp-4.15-lp-interop-jws-interop-ibmcloud

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-jws-qe-interop-ocp-ci-main-ocp-4.14-lp-interop-jws-interop-ibmcloud periodic-ci-jws-qe-interop-ocp-ci-main-ocp-4.15-lp-interop-jws-interop-ibmcloud

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-jws-qe-interop-ocp-ci-main-ocp-4.15-lp-interop-jws-interop-ibmcloud

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-jws-qe-interop-ocp-ci-main-ocp-4.14-lp-interop-jws-interop-ibmcloud periodic-ci-jws-qe-interop-ocp-ci-main-ocp-4.15-lp-interop-jws-interop-ibmcloud

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-jws-qe-interop-ocp-ci-main-ocp-4.14-lp-interop-jws-interop-aws periodic-ci-jws-qe-interop-ocp-ci-main-ocp-4.15-lp-interop-jws-interop-aws

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-jws-qe-interop-ocp-ci-main-ocp-4.14-lp-interop-jws-interop-ibmcloud periodic-ci-jws-qe-interop-ocp-ci-main-ocp-4.15-lp-interop-jws-interop-ibmcloud

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-jws-qe-interop-ocp-ci-main-ocp-4.14-lp-interop-jws-interop-ibmcloud periodic-ci-jws-qe-interop-ocp-ci-main-ocp-4.15-lp-interop-jws-interop-ibmcloud

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-jws-qe-interop-ocp-ci-main-ocp-4.14-lp-interop-jws-interop-ibmcloud periodic-ci-jws-qe-interop-ocp-ci-main-ocp-4.15-lp-interop-jws-interop-ibmcloud

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-jws-qe-interop-ocp-ci-main-ocp-4.14-lp-interop-jws-interop-ibmcloud periodic-ci-jws-qe-interop-ocp-ci-main-ocp-4.15-lp-interop-jws-interop-ibmcloud

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-jws-qe-interop-ocp-ci-main-ocp-4.15-lp-interop-jws-interop-ibmcloud periodic-ci-jws-qe-interop-ocp-ci-main-ocp-4.16-lp-interop-jws-interop-ibmcloud

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-jws-qe-interop-ocp-ci-main-ocp-4.15-lp-interop-jws-interop-ibmcloud periodic-ci-jws-qe-interop-ocp-ci-main-ocp-4.16-lp-interop-jws-interop-ibmcloud

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-jws-qe-interop-ocp-ci-main-ocp-4.15-lp-interop-jws-interop-ibmcloud periodic-ci-jws-qe-interop-ocp-ci-main-ocp-4.16-lp-interop-jws-interop-ibmcloud

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-jws-qe-interop-ocp-ci-main-ocp-4.16-lp-interop-jws-interop-ibmcloud

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-jws-qe-interop-ocp-ci-main-ocp-4.15-lp-interop-jws-interop-ibmcloud periodic-ci-jws-qe-interop-ocp-ci-main-ocp-4.16-lp-interop-jws-interop-ibmcloud

File: ci-operator/config/jws-qe/interop-ocp-ci/jws-qe-interop-ocp-ci-main__ocp-4.14-lp-interop.yaml [hunk 1]
```diff
@@ -43,6 +40,7 @@ tests:
               {"jira_project": "CSPIT", "jira_additional_labels": ["!default"]}
```
Comment (calebevans) [inline ci-operator/config/jws-qe/interop-ocp-ci/jws-qe-interop-ocp-ci-main__ocp-4.14-lp-interop.yaml:40]: We can remove the success rules as it is already defined in https://raw.githubusercontent.com/CSPI-QE/cspi-utils/main/firewatch-base-configs/aws-ipi/lp-interop.json 

Should be able to remove it from all the configs in this PR

**Review by calebevans — CHANGES_REQUESTED** [review]

**Review by Simranpal — APPROVED** [review]
/lgtm

**Review by calebevans — APPROVED** [review]
/lgtm

**Pull request merged**